test/suite/helper.py

```
@@ -118,9 +118,9 @@ def copy_wiredtiger_home(testcase, olddir, newdir, aligned=True):
     for fname in os.listdir(olddir):
         fullname = os.path.join(olddir, fname)
         # Skip lock file, on Windows it is locked.
-        # Skip temporary log files.
+        # Skip all temporary log and live restore files.
         if os.path.isfile(fullname) and "WiredTiger.lock" not in fullname and \
-            "WiredTigerTmplog" not in fullname and \
+            "tmp" not in fullname.lower() and \
```

I suspect this is too generic as a developer could get unlucky and decide to create `file:ATMPetrel.wt` which will be silently ignored. There's no examples of this in `test/suite` but I'd prefer explicit fully matched strings when we silently ignore files.

---

Yeah that's fair

---

I removed it because there's no way to tell if background migration has started on the bulk file or not. After the restart we could try an open a bulk cursor which may or may not work at that point.

---

Reverted back to the original `WiredTigerTmplog` now that test scenario is removed https://github.com/wiredtiger/wiredtiger/pull/11780/commits/c5d281f54c2f792dee3e42939ea14fb60abdd9e0